src/pages/blocked.tsx

```diff
@@ -0,0 +1,11 @@
+import { ErrorBoundary } from '../components/errors/ErrorBoundary';
+
+export default function Page() {
+  return (
+    <ErrorBoundary>
+      {(() => {
+        throw new Error('Geo blocked');
```

Comment:
Let's add a bit more info here please. How about something like "Your region has been blocked from accessing this service due to OFAC compliance requirements"